src/content/docs/release-notes/docs-release-notes/docs-6-7-2024.mdx

```diff
@@ -0,0 +1,52 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-07'
+version: 'ersion: May 31-June 6, 2024'
+---
+
+### New docs
+
+* A new [OpenTelemetry data in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview) doc is an index to help you find the OpenTelemetry info you're looking for.
+
+### Major changes
+
+* We've combined all our [.NET API](/docs/apm/agents/net-agent/net-agent-api/net-agent-api) docs so now you can see them all in one place.
+* Consolidated Java and Python APM info into [Monitor APM apps and services with OpenTelemetry](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro) so you can get this info in one place.
+
+
+### Minor changes
+
+* Updated [PHP agent operating system support](/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements/#operating-systems) to state that we support an OS version one year after a vendor's end-of-life date. (This saves us some maintenance overhead, because we don't need to list specific versions now.)
+* Our .NET APM agent for .NET Core and Framework now supports `Microsoft.Data.SqlClient` v5.2.1. Also, .NET Framework 4.6.2 or higher requires .NET APM agent version 10.0 or higher.
```

Comment:
Doesn't this overlap with the release notes? This probably doesn't hurt since we don't have to maintain this.

Reply:
It does, but we also updated some docs around this